### PR TITLE
docs(spincli): update spincli documentation for new features

### PR DESCRIPTION
**Pull request by Chinikins**
This documents new features for the spin CLI.

For projects, these features reference:
https://github.com/spinnaker/spin/pull/261
https://github.com/spinnaker/spin/pull/260
https://github.com/spinnaker/spin/pull/258
https://github.com/spinnaker/spin/pull/270

For pipelines, these features reference:
https://github.com/spinnaker/spin/pull/280
https://github.com/spinnaker/spin/pull/278

While these features are in current master, I'm unsure of how coordination with spin cli releases and documentation get coordinated, but thought I'd open the PR since its in master for spin CLI.

### CODE REVIEW

File: guides/spin/projects/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Overview
+
+Once you have `spin` installed and configured, you can use it to start
+managing your Spinnaker pipelines.
```
Comment (317brian) [inline guides/spin/projects/index.md]: This should be `managing your Spinnaker projects` shouldn't it?

File: guides/spin/pipeline/index.md [hunk 2]
```diff
@@ -120,10 +120,30 @@ flag or via STDIN, e.g.
...
 
 Pipeline deleted
 ```
+
+### Enable/Disable a pipeline with `update`
+
+```bash
+spin pipeline update --application my-app --name my-pipeline --enabled
+
+Pipeline updated
```
Comment (317brian) [inline guides/spin/pipeline/index.md:144]: `spin` returns `Pipeline updated` when the command is successful I'm assuming? Does it return an error if it fails?

Maybe add 
```
# Returns the following message if the pipeline gets updated successfully.

File: guides/spin/pipeline/index.md [hunk 3]
```diff
@@ -120,10 +120,30 @@ flag or via STDIN, e.g.
...
+```
+
+```bash
+spin pipeline update --application my-app --name my-pipeline --disabled
+
+Pipeline updated
```
Comment (317brian) [inline guides/spin/pipeline/index.md:151]: Same as above

File: guides/spin/pipeline/index.md [hunk 4]
```diff
@@ -120,10 +120,30 @@ flag or via STDIN, e.g.
...
+
 ### Delete a pipeline with `delete`
 
 ```bash
 spin pipeline delete --name my-pipeline --application my-app
 
 Pipeline deleted
 ```
+
+### Enable/Disable a pipeline with `update`
```
Comment (317brian) [inline guides/spin/pipeline/index.md]:
```suggestion
### Enable/disable a pipeline with `update`
```

File: guides/spin/projects/index.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  -k, --insecure                 ignore certificate errors
+      --no-color                 disable color (default true)
+  -o, --output string            configure output formatting
+  -q, --quiet                    squelch non-essential output
+
+Use " project [command] --help" for more information about a command.
+```
+
+The following assumes Spinnaker is running and Gate is
+listening on `http://localhost:8084`. If gate is running elsewhere,
```
Comment (317brian) [inline guides/spin/projects/index.md]: Capital `G` on Gate.

File: guides/spin/projects/index.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+you can set the Gate endpoint with the global `--gate-endpoint` flag.
+
+## Managing Your Pipeline's Lifecycle
```
Comment (317brian) [inline guides/spin/projects/index.md]:
```suggestion
## Managing your pipeline's lifecycle
```

File: guides/spin/projects/index.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+### Create and update pipelines with `save`
+
+```bash
+$ spin project save --file <path to project json>
+
+Parsed submitted project: <...>
```
Comment (317brian) [inline guides/spin/projects/index.md]: What gets returned in `<...>`?

**Review by 317brian — CHANGES_REQUESTED** [review]
Thanks for documenting the work you did! I have a few specific comments to address. Some general suggestions:
* Add comments in the CLI examples since this is brand new functionality. 
* The new page needs to be added to `_data/navigation.yml` so that it appears in the navigation. Currently the page is only accessible if I go to the page URL directly:
image
There's already an existing section w/ the other `spin` pages, so you should be able to just copy those.

**Review by 317brian — APPROVED** [review]
LGTM. I'm not sure when the spincli version with your changes will be released, so please coordinate with the release manager @kevinawoo on when to merge the docs. 

Thanks for working on this!

**Review by 317brian — APPROVED** [review]
LGTM. I'm not sure when the spincli version with your changes will be released, so please coordinate with the release manager @kevinawoo on when to merge the docs. 

Thanks for working on this!

**Comment by kevinawoo** [comment]
Those new features are for 1.22, but they're on `master` right now. @brian-armory should we only merge it after 1.22 has been released?

[Here's the calendar](https://spinnaker.io/community/releases/release-cadence/#upcoming-releases). It looks like it's suppose to go out around 8/24.

**Comment by 317brian** [comment]
Cool, let's merge this when it's available in a release on Aug 24.

**Comment by Chinikins** [comment]
@kevinawoo  @brian-armory  can this get merged now that @german-muzquiz has released spin 1.18.0? :tada:

**Comment by 317brian** [comment]
I'll merge it now. Thanks for following up @Chinikins!

**Pull request merged**